src/workloads/execution/SSBColumnStoreIndex.yml

```diff
@@ -0,0 +1,595 @@
+    explain: *Q1_2Query
+    verbosity:
+      executionStats
+- OperationMetricsName: Q1_3Query
+  OperationName: RunCommand
+  OperationCommand:
+    explain: *Q1_3Query
+    verbosity:
+      executionStats
+- OperationMetricsName: Q2_1Query
+  OperationName: RunCommand
+  OperationCommand:
+    explain: *Q2_1Query
+    verbosity:
+      executionStats
+- OperationMetricsName: Q2_2Query
+  OperationName: RunCommand
+  OperationCommand:
+    explain: *Q2_2Query
+    verbosity:
+      executionStats
+- OperationMetricsName: Q2_3Query
+  OperationName: RunCommand
+  OperationCommand:
+    explain: *Q2_3Query
+    verbosity:
+      executionStats
+- OperationMetricsName: Q3Query
+  OperationName: RunCommand
+  OperationCommand:
+    explain: *Q3Query
+    verbosity:
+      executionStats
+- OperationMetricsName: Q4_1Query
+  OperationName: RunCommand
+  OperationCommand:
+    explain: *Q4_1Query
+    verbosity:
+      executionStats
+- OperationMetricsName: Q4_2Query
+  OperationName: RunCommand
+  OperationCommand:
+    explain: *Q4_2Query
+    verbosity:
+      executionStats
+- OperationMetricsName: Q4_3Query
+  OperationName: RunCommand
+  OperationCommand:
+    explain: *Q4_3Query
+    verbosity:
+      executionStats
+
+# Defines how many times each test operation is executed.
+TestRepeatCount: &TestRepeatCount 1
+
+SetupPhase: &SetupPhase 0
+BuildColumnStoreIndexPhase: &BuildColumnStoreIndexPhase 1
+ColumnstoreScanPhase: &ColumnstoreScanPhase 2
+DropColumnstoreIndexPhase: &DropColumnstoreIndexPhase 3
+CollScanPhase: &CollScanPhase 4
+MaxPhases: &MaxPhases 5
+
+ColumnStoreIndexName: &ColumnStoreIndexName $**_columnstore
+
+Actors:
+- Name: Setup
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [*SetupPhase]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Thread: 1
+        Database: admin
+        Operations:
+        - OperationName: AdminCommand
+          OperationCommand:
+            setParameter: 1
+            internalQueryFrameworkControl: "trySbeEngine"
+
+- Name: BuildColumnStoreIndex
+  Type: RunCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [*BuildColumnStoreIndexPhase]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Thread: 1
+        Database: *db
+        Operations:
+        - OperationMetricsName: BulkBuildColumnStoreIndex
+          OperationName: RunCommand
+          OperationCommand:
+            createIndexes: *Collection
+            indexes:
+            - key: {"$**": "columnstore"}
+              name: *ColumnStoreIndexName
+
+# The 'ColumnstoreScan' actor measures the perf with columnar indexes.
+- Name: ColumnstoreScan
+  Type: RunCommand
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [*ColumnstoreScanPhase]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *TestRepeatCount
+        Database: *db
+        Operations: *TestOperations
```

Comment:
I presume this will run the operations in the order they are listed with no cleanup/reset of the actor between them, right? That is, the caching pattern will depend on the order of the queries. Whether that matters of not, I have no idea

Reply:
That's correct. My assumption is that with a data set larger than the size of RAM, a warm cache won't matter much.

Reply:
As I noted earlier, I suspect you would be better off to run them separately.

Reply:
What would it take in terms of the task's run-time to run the queries separately? If we run two patches with the queries ordered differently and see no significant difference, would that be sufficient "proof" of the expectation that warm cache doesn't matter much?

Reply:
Genny workloads collect measurements such as disk utilization (%) and reads/writes per second (mean, max, p99). The measurements are collected per phase, so if we split each query to run in its own phase (with or without explicit cleanup in between) at least we can observe some aspects of disk I/O behaviour? And I filed PERF-3535 to collect page faults which would be a better metric for our purposes.